org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -279,6 +295,21 @@ protected override void Reset()
             ApplyRequiredSettings();
         }
 
+        /// <summary>
+        /// A Unity event function that is called when the script is loaded.
+        /// </summary>
+        private void Start()
```

Comment:
To match the other Unity functions in this file, as well as to prevent derived classes from having their calls blocked, it would be good to make this `protected` and probably `virtual`.

For more info: https://github.com/microsoft/Microsoft.Unity.Analyzers/blob/main/doc/UNT0021.md

A downside to this change is it'd _technically_ be a breaking change if anybody has created their own derived classes with a defined `Start()` method. It might be worth discussing how we want to approach this.

Reply:
You are right, thanks for the feedback.  Updated in latest commit.